lioncore/metametamodel.adoc

```diff
@@ -223,53 +225,11 @@ An EnumerationLiteral is similar to Ecore’s `EEnumLiteral` and to MPS’ `Enum
 .Characteristics
 An EnumerationLiteral has a <<NamespacedEntity.name, `name`>> and an <<NamespacedEntity.key, `key`>>.
 
-An Enumeration is a <<NamespacedEntity>> (as it has an identity and it is contained in the namespace of its <<Enumeration>>).
+An Enumeration is a <<NamespacedEntity>> (as it has an identity, and it is contained in the namespace of its <<Enumeration>>).
 
 .Constraints
 TBD
 
-[[Typedef]]
```

Comment:
Maybe we could open two issues, one for typedef and one for annotation and copy the text regarding annotation in the corresponding issue

Reply:
Typedefs: #20 
Annotations: #13